lib/map/map_world.dart

```diff
@@ -161,4 +157,104 @@ class MapWorld extends MapGame {
       canvas.drawPath(path, _paintPath);
     }
   }
+
+  void _updateTilesToRender() async {
```

Comment:
Actually this is a biggest performance bottleneck. is very visible running on the web

Reply:
It seems to be something of a web debug way. The crash is caused by `Future.forEach`. but running in release mode everything OK

Reply:
@RafaelBarbosatec I am testing both on Android and Web and it seems very nice, the performance issue on web is solved running in release mode so I don't think it is a big problem.

The only thing I noticed is that the player movement is lagging if I increase his speed (I am using 300.0), take a look:
https://imgur.com/a/GVtKHwI

Reply:
Perfect. Thanks so much!
 I'll take a look at that case.

Reply:
Hi @KyleKun !
I found the problem. I had to make improvements to the collision system!
Try again and tell me what you think.